### PR TITLE
New version: KuaiFan.DooTask version 0.39.97

### PR DESCRIPTION
**Pull request by spectopo**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings) in workflow run [#9473](https://github.com/SpecterShell/Dumplings/actions/runs/11782091117).

<details>

<summary>Log</summary>

````
Updated: 0.39.88 → 0.39.97
Submitting WinGet manifests
Creating manifests
Uploading manifests and making commits
Creating a pull request
````

</details>

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/190458)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -2145844844
> 2024-11-11 08:26:24.861 [CORE] DeliveryOptimization error: 0x80190194, extended error: 0x00000000
> 2024-11-11 08:26:24.899 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(289)\WindowsPackageManager.dll!00007FFBA3CCA465: (caller: 00007FFBA3CCAD01) Exception(1) tid(414) 80190194 Not found (404).
> 2024-11-11 08:26:25.453 [CORE] Download request failed. Returned status: 404
> 2024-11-11 08:26:25.453 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(148)\WindowsPackageManager.dll!00007FFBA3CC43D6: (caller: 00007FFBA3CC5C49) Exception(2) tid(414) 80190194 Not found (404).
> 2024-11-11 08:26:25.622 [CLI ] Failed to download, waiting a bit and retry. Url: https://github.com/kuaifan/dootask/releases/download/v0.39.97/DooTask-v0.39.97-win-x64.exe
> 2024-11-11 08:26:27.924 [CORE] DeliveryOptimization error: 0x80190194, extended error: 0x00000000
> 2024-11-11 08:26:27.924 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(289)\WindowsPackageManager.dll!00007FFBA3CCA465: (caller: 00007FFBA3CCAD01) Exception(3) tid(414) 80190194 Not found (404).
> 2024-11-11 08:26:28.021 [CORE] Download request failed. Returned status: 404
> 2024-11-11 08:26:28.021 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(148)\WindowsPackageManager.dll!00007FFBA3CC43D6: (caller: 00007FFBA3CC5C49) Exception(4) tid(414) 80190194 Not found (404).
> 2024-11-11 08:26:21.095 [CORE] DeliveryOptimization error: 0x80190194, extended error: 0x00000000
> 2024-11-11 08:26:21.130 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(289)\WindowsPackageManager.dll!00007FFBB3DFA465: (caller: 00007FFBB3DFAD01) Exception(1) tid(1998) 80190194 Not found (404).
> 2024-11-11 08:26:21.763 [CORE] Download request failed. Returned status: 404
> 2024-11-11 08:26:21.763 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(148)\WindowsPackageManager.dll!00007FFBB3DF43D6: (caller: 00007FFBB3DF5C49) Exception(2) tid(1998) 80190194 Not found (404).
> 2024-11-11 08:26:22.014 [CLI ] Failed to download, waiting a bit and retry. Url: https://github.com/kuaifan/dootask/releases/download/v0.39.97/DooTask-v0.39.97-win-x64.exe
> 2024-11-11 08:26:24.330 [CORE] DeliveryOptimization error: 0x80190194, extended error: 0x00000000
> 2024-11-11 08:26:24.330 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(289)\WindowsPackageManager.dll!00007FFBB3DFA465: (caller: 00007FFBB3DFAD01) Exception(3) tid(1998) 80190194 Not found (404).
> 2024-11-11 08:26:24.397 [CORE] Download request failed. Returned status: 404
> 2024-11-11 08:26:24.397 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(148)\WindowsPackageManager.dll!00007FFBB3DF43D6: (caller: 00007FFBB3DF5C49) Exception(4) tid(1998) 80190194 Not found (404).

Automated error analysis suggests -2145844844 may mean Not Found (404).

Automated error analysis suggests 80190194 may mean Not found (404).

(Automated response - build 897.)

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

File: manifests/k/KuaiFan/DooTask/0.39.97/KuaiFan.DooTask.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  Scope: machine
+  InstallerUrl: https://github.com/kuaifan/dootask/releases/download/v0.39.97/DooTask-v0.39.97-win-x64.exe
+  InstallerSha256: 52BBC172D920F347BBA814D8151DA900BF6FB1FC32A101350E0B1AA4195DED28
+  InstallerSwitches:
+    Custom: /allusers
+  ProductCode: 9a445616-2838-509d-bd29-65c401183c4f
+- Architecture: arm64
+  Scope: user
+  InstallerUrl: https://github.com/kuaifan/dootask/releases/download/v0.39.97/DooTask-v0.39.97-win-arm64.exe
+  InstallerSha256: 154E231F60A37FEA425C1209FAFB4F36DDCCEFA754D5B94150C3F91E09EE266E
```
Comment (stephengillie) [inline manifests/k/KuaiFan/DooTask/0.39.97/KuaiFan.DooTask.installer.yaml:29]:
```suggestion
  InstallerSha256: 3FE97FD8DDE28D7F3CC6B28CBAA83D6C9C7C590775C717E52CA8494B4DEE7D82
```

File: manifests/k/KuaiFan/DooTask/0.39.97/KuaiFan.DooTask.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  InstallerSwitches:
+    Custom: /currentuser
+  ProductCode: 9a445616-2838-509d-bd29-65c401183c4f
+- Architecture: arm64
+  Scope: machine
+  InstallerUrl: https://github.com/kuaifan/dootask/releases/download/v0.39.97/DooTask-v0.39.97-win-arm64.exe
+  InstallerSha256: 154E231F60A37FEA425C1209FAFB4F36DDCCEFA754D5B94150C3F91E09EE266E
```
Comment (stephengillie) [inline manifests/k/KuaiFan/DooTask/0.39.97/KuaiFan.DooTask.installer.yaml:36]:
```suggestion
  InstallerSha256: 3FE97FD8DDE28D7F3CC6B28CBAA83D6C9C7C590775C717E52CA8494B4DEE7D82
```

File: manifests/k/KuaiFan/DooTask/0.39.97/KuaiFan.DooTask.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+InstallerType: nullsoft
+InstallerSwitches:
+  Upgrade: --updated
+UpgradeBehavior: install
+ReleaseDate: 2024-11-11
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/kuaifan/dootask/releases/download/v0.39.97/DooTask-v0.39.97-win-x64.exe
+  InstallerSha256: 52BBC172D920F347BBA814D8151DA900BF6FB1FC32A101350E0B1AA4195DED28
```
Comment (stephengillie) [inline manifests/k/KuaiFan/DooTask/0.39.97/KuaiFan.DooTask.installer.yaml:15]:
```suggestion
  InstallerSha256: FE024A77D49A940B0537E588D4976B244E1B19C20F070C36952D81B30D3AFCAB
```

File: manifests/k/KuaiFan/DooTask/0.39.97/KuaiFan.DooTask.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  InstallerSwitches:
+    Custom: /currentuser
+  ProductCode: 9a445616-2838-509d-bd29-65c401183c4f
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/kuaifan/dootask/releases/download/v0.39.97/DooTask-v0.39.97-win-x64.exe
+  InstallerSha256: 52BBC172D920F347BBA814D8151DA900BF6FB1FC32A101350E0B1AA4195DED28
```
Comment (stephengillie) [inline manifests/k/KuaiFan/DooTask/0.39.97/KuaiFan.DooTask.installer.yaml:22]:
```suggestion
  InstallerSha256: FE024A77D49A940B0537E588D4976B244E1B19C20F070C36952D81B30D3AFCAB
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
> 11/12/2024 4:09:27 AM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   3FE97FD8DDE28D7F3CC6B28CBAA83D6C9C7C590775C717E52CA8494B4DEE7D82
Expected hash: 154e231f60a37fea425c1209fafb4f36ddccefa754d5b94150c3f91e09ee266e
Specified hash doesn't match.
Actual hash:   FE024A77D49A940B0537E588D4976B244E1B19C20F070C36952D81B30D3AFCAB
Expected hash: 52bbc172d920f347bba814d8151da900bf6fb1fc32a101350e0b1aa4195ded28

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**